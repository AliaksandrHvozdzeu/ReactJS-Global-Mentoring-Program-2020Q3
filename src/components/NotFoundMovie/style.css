.movie-no-results {
  display: flex;
  justify-content: center;
  align-items: center;
  font-size: 3rem;
  color: white;
  height: calc(56vh - 4rem);
  font-weight: 100;
}
