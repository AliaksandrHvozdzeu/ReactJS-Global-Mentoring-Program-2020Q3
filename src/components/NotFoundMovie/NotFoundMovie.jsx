import React from 'react';
import './style.css';

export default function NotFoundMovie() {

  return (
    <>
      <section className="movie-no-results">
        No Movie Found
      </section>
    </>
  );

}
