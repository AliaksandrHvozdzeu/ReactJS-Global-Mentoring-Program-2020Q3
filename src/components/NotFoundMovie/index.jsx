import React from 'react';
import './style.css';

const NotFoundMovie = () => (
  <>
    <section className="movie-no-results">
      No Movie Found
    </section>
  </>
);

export default NotFoundMovie;
