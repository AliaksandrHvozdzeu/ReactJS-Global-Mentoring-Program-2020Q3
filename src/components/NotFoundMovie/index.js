import NotFoundMovie from './js/NotFoundMovie';

export default NotFoundMovie;
