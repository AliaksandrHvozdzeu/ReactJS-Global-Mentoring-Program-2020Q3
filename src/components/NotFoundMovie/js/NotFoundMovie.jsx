import React from 'react';
import '../css/NotFoundMovie.css';

export default function NotFoundMovie() {

  return (
    <>
      <section className="movie-no-results">
        No Movie Found
      </section>
    </>
  );

}
