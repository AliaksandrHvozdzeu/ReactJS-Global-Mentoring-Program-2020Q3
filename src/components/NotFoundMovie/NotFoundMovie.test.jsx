import React from 'react';
import rerender from 'react-test-renderer';
import NotFoundMovie from './js/NotFoundMovie';

describe('when NotFoundMovie', () => {

  test('then snapshot created', () => {
    const component = rerender.create(<NotFoundMovie/>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});
