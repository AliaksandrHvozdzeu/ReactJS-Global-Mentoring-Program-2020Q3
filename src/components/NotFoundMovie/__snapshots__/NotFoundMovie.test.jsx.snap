// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`when NotFoundMovie then snapshot created 1`] = `
<section
  className="movie-no-results"
>
  No Movie Found
</section>
`;
